bookwyrm/models/author.py

```diff
@@ -18,6 +18,9 @@ class Author(BookDataModel):
     wikipedia_link = fields.CharField(
         max_length=255, blank=True, null=True, deduplication_field=True
     )
+    wikidata = fields.CharField(
```

Comment:
You don't need to add this here -- the field is already on this model, it's inherited from the parent model, `BookDataModel`: https://github.com/bookwyrm-social/bookwyrm/blob/main/bookwyrm/models/book.py#L55